portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsSettings.testcase

```diff
@@ -180,6 +180,78 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: LRAC-10747 | Automation ID: LRAC-11378 | Test Summary: Delete Custom Events description"
```

Comment:
Hi @ViniLopes20 since this testcase is based on a bug, I think the description should be like this (as I'm seeing in other testcases)
``@description = "Bug: LRAC-10747 | Automation ID: LRAC-11378 | Test Summary: Delete Custom Events description"``

Reply:
Yesss, @manuferreira. I forgot to put Bug at the beginning. Thanks for observing this 🦸🏽‍♀️